packages/apify/src/proxy_configuration.ts

```diff
@@ -418,52 +418,47 @@ export class ProxyConfiguration extends CoreProxyConfiguration {
      * Checks if Apify Token is provided in env and gets the password via API and sets it to env
      */
     protected async _setPasswordIfToken(): Promise<void> {
-        const token = this.config.get('token');
+        if (!this.password) {
```

Comment:
early return is better than this, now the whole method implementation is inside the if block.

```suggestion
        if (this.password) {
            return;
        }
```

Reply:
Changed.